src/pages/migrating/guide/develop.mdx

```diff
@@ -17,24 +17,23 @@ Step-by-step guide to updating your code from Carbon v10 to v11.
 <AnchorLinks>
 
 <AnchorLink>Overview</AnchorLink>
-<AnchorLink>carbon-components-react</AnchorLink>
-<AnchorLink>carbon-components</AnchorLink>
-<AnchorLink>@carbon/icons-react</AnchorLink>
-<AnchorLink>carbon-icons</AnchorLink>
-<AnchorLink>Elements</AnchorLink>
+<AnchorLink>Migrating a React app</AnchorLink>
+<AnchorLink>Migrating an app using just styles</AnchorLink>
+<AnchorLink>Migrating an app using Angular, Svelte, Vue, etc.</AnchorLink>
+<AnchorLink>Migrating a library</AnchorLink>
```

Comment:
These last three don't seem to be mapped to any headings

Reply:
yeah, we haven't written those sections yet ;)